### PR TITLE
Feature/web 48 add speaker detail

### PR DESCRIPTION
**Pull request by uncenteredDiv**

### CODE REVIEW

File: src/utils/constants.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
-export const SPONSOR_LIST_FILTER_TAG_KEY = "FRONT24";
+export const SPONSOR_LIST_FILTER_TAG_KEY = "FRONT22";
```
Comment (backflip) [inline src/utils/constants.ts]: Could we change all these to `FRONNT24` so we can test the proper content?

Comment (backflip) [reply src/utils/constants.ts]: Ah, just saw the other PR. 👍

File: src/components/TalkDetail.astro [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+const { talk } = Astro.props;
+---
+
+<article class="talk">
+	<h3>{talk.title}</h3>
+	<div class="meta">
+		<div class="metaGroup">
+			<div class="talkType typo-body-caption">Talk</div>
+			<div>{formatDate(talk.date)}</div>
```
Comment (backflip) [inline src/components/TalkDetail.astro]: As we don't know the day yet for talks, this would have to be optional, too.

Comment (uncenteredDiv) [reply src/components/TalkDetail.astro]: done

File: src/utils/constants.ts (hunk 1 above)
Comment (backflip) [reply src/utils/constants.ts]: It probably makes sense to merge the other one first and then update this one, right?

Comment (uncenteredDiv) [reply src/utils/constants.ts]: yeah - done

Comment (backflip) [reply src/utils/constants.ts]: LGTM after rebasing. 👍 (I won't be able to give any feedback later today or tomorrow)

**Review by backflip — APPROVED** [review]
https://frontconference-website-git-feat-f3d779-front-conference-zurich.vercel.app/speakers/ryan-townsend looks great to me!

**Pull request merged**